color_detection/points[] pixels
---
geometry_msgs/Point[] conePos
